coding/src/reed_solomon/mod.rs

```diff
@@ -448,6 +575,11 @@ mod tests {
         let (root, chunks) = encode::<Sha256>(total, min, data.to_vec()).unwrap();
         assert_eq!(chunks.len(), total as usize);
 
+        // Verify all chunks
```

Comment:
Realized there were no explicit tests on proofs (although they are checked in decode).